### PR TITLE
Add nft support - fix firewalld on TW (which will fix nftables as well)

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/80966
Ticket: https://progress.opensuse.org/issues/89999
Verification TW: http://kazhua.qa.suse.de/tests/307
Verification SLE: http://kazhua.qa.suse.de/tests/308

### CODE REVIEW

**Review by punkioudi — APPROVED** [review]
lgtm!

File: tests/console/firewalld.pm [hunk 1]
```diff
@@ -27,59 +27,82 @@ sub pre_test {
...
-        assert_script_run("iptables -C IN_public_allow -p udp --dport 2000:3000 -m conntrack --ctstate NEW -j ACCEPT");
-    }
+    check_rules;
 
     # Reload default configuration
     record_info 'Reload default configuration';
     assert_script_run("firewall-cmd --reload");
-    assert_script_run("if [ `iptables -L IN_public_allow --line-numbers | sed '/^num\\|^\$\\|^Chain/d' | wc -l` -eq `cat /tmp/nr_rules.txt` ]; then /usr/bin/true; else /usr/bin/false; fi");
+    if (!is_tumbleweed) {
+        assert_script_run("if [ `iptables -L IN_public_allow --line-numbers | sed '/^num\\|^\$\\|^Chain/d' | wc -l` -eq `cat /tmp/nr_rules.txt` ]; then /usr/bin/true; else /usr/bin/false; fi");
```
Comment (pdostal) [inline tests/console/firewalld.pm:87]:
```suggestion
        assert_script_run("[ `iptables -L IN_public_allow --line-numbers | sed '/^num\\|^\$\\|^Chain/d' | wc -l` -eq `cat /tmp/nr_rules.txt` ]");
```

Comment (asdil12) [reply tests/console/firewalld.pm:87]: Good idea - I will handle that in another PR

Comment (asdil12) [reply tests/console/firewalld.pm:87]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12277

**Pull request merged**